test/specs/test.spec.js

```diff
@@ -0,0 +1,75 @@
+import mainPage from "../pageobjects/main.page";  
+import loginPage from "../pageobjects/login.page";
+import checkPage from "../pageobjects/check.page";
+import rawPage from "../pageobjects/raw.page";
+
+const id ={
+    login : 'atbinjs',
+    email : 'atbinjs@yopmail.com',
+    password : 'superTestPassword',
+}
+describe('check bin in Pastebin',() => {
+    /* I wanted to have more tries. But I had not found solution to pass "Проверка безопасности подключения к сайту"*/
+    // it ('login', async () => {
+    //     await mainPage.open();
+    //     await loginPage.open();
+    //     //await loginPage.inputPassword.waitForDisplayed({timeout:60000, interval:10000}); //try to wait "Проверка безопасности подключения к сайту"
+    //     await loginPage.login(id.login, id.password);
+    // })   
+    
+    it('I can win', async () =>{
+        //open site
+        await mainPage.open();
+        //input message
+        await mainPage.postText.setValue("Hello from WebDriver");
+        //choose Expiration
+        await mainPage.setExpiration('10 Minutes');
+        //input title
+        await mainPage.postName.setValue("Hello web");
+        //send message
+        await mainPage.create();
```

Comment:
This sequence can be wrapped into separate method like `createPaste(parameters)`.